emulators/qemu/Portfile

```diff
@@ -44,8 +47,8 @@ depends_lib             port:curl \
 compiler.blacklist      {clang < 500} *gcc-4.2 *gcc-4.0 gcc-3.3
 if {${os.platform} eq "darwin" && ${os.major} < 11} {
     # Emulated TLS needed, which was added in LLVM 3.8
-    compiler.blacklist-append   macports-clang-3.4 macports-clang-3.3
-    compiler.fallback-append    macports-clang-8.0
+    compiler.blacklist-append   macports-clang-3.4 macports-clang-3.3 macports-clang-3.7
+    compiler.fallback-append    macports-clang-8.0 macports-clang-7.0 macports-clang-6.0 macports-clang-5.0
```

Comment:
The changes to these two lines do nothing. Clang 3.7 was never in the fallback list, so blacklisting it is unnecessary. All the compilers listed after macports-clang-8.0 in compiler.fallback will never be used, because macports-clang-8.0 is always available (since it's an installable port) and comes earlier in the list.

Reply:
Thanks for taking the time to feedback. I admit I just filled in the rest of the supported compilers per their capabilities while I was in the neighborhood. People ask for these ports on 10.4 and 10.5, and sometimes I find having the list complete is useful. I understand this will all become simpler with the more detailed compiler specing coming in MP 3.6+.